kf/bpf.go

```diff
@@ -219,6 +221,7 @@ func StopExternalRunningProcess(processName string) error {
 }
 
 // Stop returns the last error seen, but stops bpf program.
+// Stops the user programs if any, and unloads the kernel program.
```

Comment:
```suggestion
// Stops the user programs if any, and unloads the BPF program.
```
Don't use the term "kernel program", use "BPF program" for consistency with the rest of the docs. (Also because some platform might support BPF programs running in a user space environment.)